README.md

```diff
@@ -478,12 +478,12 @@ iterator debugging.
 ### Android ndk-build
 
 SPIR-V Tools supports building static libraries `libSPIRV-Tools.a` and
-`libSPIRV-Tools-opt.a` for Android:
+`libSPIRV-Tools-opt.a` for Android.  Using the Android NDK r25c or later:
```

Comment:
FYI.  Before NDK r25, the ndk-build process would default to the prebuilt Python under `$NDK_HOME/prebuilts/<host-os-and-arch>/bin/python2.7`

Under NDK r25, the default Python is 
`$NDK_HOME/toolchains/llvm/prebuilt/<host-os-and-arch>/python3`

Python 2.6 does not support XML parsing with UTF-8 encoding, as far as I can tell.